### PR TITLE
City events crud

### PR DESCRIPTION
**Pull request by GermanAlexis**

### CODE REVIEW

File: app/Repository/City/CityEventRepositoryInterface.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+
+namespace App\Repository\City;
+
+interface CityEventRepositoryInterface
+{
+    public function storecityevents($data); // esto a donde direccion?
```
Comment (CamiloSerrano95) [inline app/Repository/City/CityEventRepositoryInterface.php]: Este método debe ir CityRepositoryIterface

File: app/Http/Controllers/CityController.php [hunk 2]
```diff
@@ -62,4 +63,61 @@ public function update($cityId, Request $request){
...
+    public function allevents(){
+        $event = $this->cityRepository->allevents();
+        $response = [
+            'success' => 1,
+            'data' => $event
+        ];
+        return response()->json($response, 200);
+    }
+
+    public function destroyevent($EventId){
```
Comment (dgarcia97) [inline app/Http/Controllers/CityController.php]: destroyEvents should be the name of the method

File: app/Http/Controllers/CityController.php [hunk 3]
```diff
@@ -62,4 +63,61 @@ public function update($cityId, Request $request){
...
+            'date' => $request->date,
+            'location' => $request->location
+        ];
+
+        $this->cityRepository->storecityevent($data);
+
+        return response()->json(['success' => 1], 200);
+    }
+
+    public function allevents(){
```
Comment (dgarcia97) [inline app/Http/Controllers/CityController.php]: allEvents should be the name of the method

File: app/Http/Controllers/CityController.php [hunk 4]
```diff
@@ -62,4 +63,61 @@ public function update($cityId, Request $request){
 
         return response()->json(['success' => 1], 200);
     }
+
+
+    // ----------  START CITY EVENTS ------------------
+    public function storecityevent(Request $request){
```
Comment (dgarcia97) [inline app/Http/Controllers/CityController.php]: storeCityEvent should be the name of the method

File: app/Repository/City/CityRepository.php [hunk 5]
```diff
@@ -42,4 +48,36 @@ public function update($cityId, $data)
...
         return true;
     }
+// ----------- Start City Events ---------------
+    public function storecityevent($data)
+    {
+        $Storeevent = $this->event->insert($data);
+        return $Storeevent;
+    }
+
+    public function allevents()
```
Comment (dgarcia97) [inline app/Repository/City/CityRepository.php]: allEvents

File: app/Repository/City/CityRepository.php [hunk 6]
```diff
@@ -42,4 +48,36 @@ public function update($cityId, $data)
...
+    {
+        return $this->event->all();
+    }
+
+    public function destroyevent($eventId)
```
Comment (dgarcia97) [inline app/Repository/City/CityRepository.php]: destroyEvent

File: app/Repository/City/CityRepositoryInterface.php [hunk 7]
```diff
@@ -10,4 +10,12 @@ public function update($cityId, $data);
     public function all();
     public function destroy($cityId);
     public function getById($cityId);
+    public function storecityevent($data);
+//-------------- Start City Event --------------
+    public function allevents();  
+    public function destroyevent($eventId);
+    public function getByeventId($eventId);
+    public function updateEvent($eventId, $data);
```
Comment (dgarcia97) [inline app/Repository/City/CityRepositoryInterface.php]: Wrong method names, check comments above

File: public/js/cityevent/storecityevent.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
+$(document).ready(function(){
+    listetableevent();
```
Comment (dgarcia97) [inline public/js/cityevent/storecityevent.js]: listTableEvent

**Pull request merged**